portfolio/src/main/webapp/photo_gallery.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Photo Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <style>
+    body{
+        background-color: #a8d0e6;
+    }
+    div.gallery {
+        margin: 5px;
+        border: 3px solid;
+        float: left;
+        width: 200px;
+    }
+    div.gallery img {
+        width: 100%;
+        height: auto;
+    }
+    div.img {
+        padding: 15px;
+        text-align: center;
+    }
+</style>
+<body>
+
+<div class="gallery">
+  <a target="_blank" href="/images/IMG_3112.JPG">
+    <img src="/images/IMG_3112.JPG" alt="Cinque Terre" width="600" height="400">
```

Comment:
In the spirit of good practices, let's give these image files more descriptive names, something like "cinque_terre.jpg"